internal/core/command/restDevice.go

```diff
@@ -96,7 +104,18 @@ func issueDeviceCommand(
 		return
 	}
 
-	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
+	// Check the header value
+	switch r.Header.Get(clients.ContentType) {
+	case clients.ContentTypeCBOR:
+		// Content-type CBOR will be sent to the caller.
+		w.Header().Set(clients.ContentType, clients.ContentTypeCBOR)
+	case clients.ContentTypeJSON:
+		// Content-type JSON will be sent to the caller.
+		w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
+	default:
+		lc.Error(fmt.Sprintf("header value neither JSON nor CBOR, but: %s", r.Header.Get(clients.ContentType)))
+	}
```

Comment:
Can we just set the response header `Content-type` to whatever the request header `Content-type` is without having to inspect it? This way we do not have to update this logic to reflect supported content types downstream. Also we can rely on the lower level functionality to fail if the content type is not supported.